pyproject.toml

```diff
@@ -17,6 +17,7 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
```

Comment:
Should 3.14 also be added to the target-version in the `[tool.black]` section below?

Reply:
Thanks @DeinAlptraum, added in `pyproject.toml` and in `check-darkgraylib.toml` as well.